x-pack/platform/plugins/shared/fleet/public/applications/fleet/sections/settings/components/edit_output_flyout/output_form_kafka_topics.tsx

```diff
@@ -73,7 +74,21 @@ export const OutputFormKafkaTopics: React.FunctionComponent<{ inputs: OutputForm
             label={
               <FormattedMessage
                 id="xpack.fleet.settings.editOutputFlyout.kafkaDynamicTopicLabel"
-                defaultMessage="Topic from field"
+                defaultMessage="Topic from field(s). For more info, see our {guideLink}"
+                values={{
+                  guideLink: (
+                    <EuiLink
+                      href="https://www.elastic.co/guide/en/beats/filebeat/current/kafka-output.html#topic-option-kafka"
```

Comment:
We'll want to use the docs links service here and register this route e.g.

https://github.com/elastic/kibana/blob/2f0bad7d39319c315597be0e464b6d41fb4c25ef/x-pack/platform/plugins/shared/fleet/public/applications/fleet/components/generate_service_token.tsx#L47

https://github.com/elastic/kibana/blob/2f0bad7d39319c315597be0e464b6d41fb4c25ef/x-pack/platform/plugins/shared/fleet/public/applications/fleet/components/generate_service_token.tsx#L54-L66

https://github.com/elastic/kibana/blob/2f0bad7d39319c315597be0e464b6d41fb4c25ef/src/platform/packages/shared/kbn-doc-links/src/types.ts#L481-L523

https://github.com/elastic/kibana/blob/2f0bad7d39319c315597be0e464b6d41fb4c25ef/src/platform/packages/shared/kbn-doc-links/src/get_doc_links.ts#L810-L854